packages/integration/src/compiler.ts

```diff
@@ -135,6 +142,28 @@ const createViteServer = async ({
   };
 };
 
+class NormalizedMap<V> extends Map<string, V> {
```

Comment:
This is the most significant change in this file. Paths are now normalised so the cache can return a value whether you provide an absolute or relative path key.